yarn-project/scripts/src/benchmarks/aggregate.ts

```diff
@@ -160,8 +160,17 @@ function processTxPXEProcessingStats(entry: TxPXEProcessingStats, results: Bench
 }
 
 /** Process entries for events tx-public-part-processed, grouped by public data writes */
-function processTxSequencerProcessingStats(entry: TxSequencerProcessingStats, results: BenchmarkCollectedResults) {
+function processTxSequencerProcessingStats(
+  entry: TxSequencerProcessingStats,
+  results: BenchmarkCollectedResults,
+  fileName: string,
+) {
   append(results, 'tx_sequencer_processing_time_ms', entry.publicDataUpdateRequests, entry.duration);
+  // only track specific txs to ensure they're doing the same thing
+  // TODO(alexg): need a better way to identify these txs
```

Comment:
Yeah it's like we want to be able to set tracing on individual transactions.

Reply:
I've created #5445